src/components/ui/source-modal/index.tsx

```
@@ -30,32 +29,137 @@ import { ScrollArea } from "../scroll-area";
 }) {
-  const { bookResponse } = useBookDetails();
-  const [isOpen, setIsOpen] = useState(false);
-  const closeMobileSidebar = useMobileReaderStore((s) => s.closeMobileSidebar);
-  const slug = useParams().bookId as string;
-  const versionId = useSearchParams().get("versionId");
-  const { copyUrl: copyShareUrl } = useBookShareUrl();
+  return (
+    <>
+      <ScrollArea className="h-96 w-full pl-5" dir="rtl">
+        <p
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)